api/helpers/things.js

```diff
@@ -219,21 +235,25 @@ function getEditXById(type) {
             }
           } else if (
             // fields that are lists of strings
-            [
-              "tags",
-              "links",
-              "images",
-              "videos",
-              "files",
-              "if_voting",
-              "evaluation_reports",
-              "evaluation_links"
-            ].includes(key)
+            ["tags", "if_voting"].includes(key)
           ) {
             updatedThingFields.push({
               key: as.name(key),
               value: as.strings(value)
             });
+          } else if (
+            // fields that are rich links
+            ["links", "images", "videos", "evaluation_links"].includes(key)
+          ) {
+            updatedThingFields.push({
+              key: as.name(key),
+              value: as.media(value)
+            });
+          } else if (["files", "evaluation_reports"]) {
```

Comment:
this list should include `photos`

Reply:
This function will go away later today when I update organizations. Methods, organizations, and cases used to all be similar enough that I tried using one function for updating them (in `things.js`) but they have diverged sufficiently, and I think it is easier to read and maintain, that having them each have their own update function makes more sense.